relentless/simulate/simulate.py

```diff
@@ -34,8 +37,8 @@ def run(self, ensemble, potentials, directory):
         ensemble : :py:class:`Ensemble`
             Simulation ensemble. Must include values for *N* and *V* even if
             these variables fluctuate.
-        potentials : :py:class:`PairMatrix`
-            Matrix of tabulated potentials for each pair.
+        potentials : :py:class:`Potentials`
+            The tabulated potentials.
```

Comment:
```suggestion
            The interaction potentials.
```
(we don't "tabulate" explicitly anymore)